Aspire Budgeting/Facilities/UserManager.swift

```diff
@@ -22,18 +22,31 @@ protocol AspireSignInInstance: AnyObject {
 
 extension GIDSignIn: AspireSignInInstance {}
 
+extension Notification.Name {
+  static let authorizerUpdated = Notification.Name("authorizerUpdated")
+}
+
+protocol AspireNotificationCenter: AnyObject {
+  func post(name aName: NSNotification.Name, object anObject: Any?, userInfo aUserInfo: [AnyHashable: Any]?)
+}
+
+extension NotificationCenter: AspireNotificationCenter {}
+
 class UserManager: NSObject, GIDSignInDelegate, ObservableObject {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mohitathwani/aspirebudgeting/blob/6ead50c19f54008e63b8e9f2ba720e23a9263087/Aspire Budgeting/Facilities/UserManager.swift#L35"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>